<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataInfo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="dataicon.qrc">
    <normaloff>:/dataicon/mainIcon.png</normaloff>:/dataicon/mainIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f8f8ff;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼매직체</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #AF1B3F;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;ASR-NER Pipeline&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="final_text">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>591</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼매직체</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;- 개체명 인식 결과 -&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="before_text">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>591</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>520</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>휴먼매직체</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: #AF1B3F;</string>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="dataicon.qrc"/>
 </resources>
 <connections/>
</ui>
